src/License.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "murmur_pch.h"
+
+#include "License.h"
+
+#include "licenses.h"
+
+QString License::license() {
+	return QString::fromUtf8(licenseMumble);
+}
+
+QString License::authors() {
+	return QString::fromUtf8(authorsMumble);
+}
+
+QList<LicenseInfo> License::thirdPartyLicenses() {
+	QList<LicenseInfo> licenses;
+	for (int i = 0; !licenses3rdParties[i].isEmpty(); i++) {
+		const ThirdPartyLicense *tpl = &licenses3rdParties[i];
+		LicenseInfo li;
+		li.name = QString::fromUtf8(tpl->name);
+		li.url = QString::fromUtf8(tpl->url);
+		li.license = QString::fromUtf8(tpl->license);
+		licenses << li;
+	}
+	return licenses;
+}
+
+QString License::printableThirdPartyLicenseInfo() {
+	QString output;
+
+	QList<LicenseInfo> thirdPartyLicenses = License::thirdPartyLicenses();
+	foreach (LicenseInfo li, thirdPartyLicenses) {
+		QString header = QString::fromLatin1("%1 (%2)").arg(li.name).arg(li.url);
```

Comment:
Personally, I think I would prefer the newlines for both header and headerHorizLine to be added in them, rather than added after-the-fact below.
Was this a deliberate choice to have the individual newlines visible below?
I think only having to add newlines for "empty lines" below would be better. The strings declared here always have a newline after them.